examples/reference/panes/Vizzu.ipynb

```diff
@@ -36,7 +36,19 @@
     "* **`animate`**: Accepts a dictionary of new 'data', 'config' and 'style' values which is used to update the chart.\n",
     "* **`stream`**: Streams new data to the plot.\n",
     "* **`patch`**: Patches one or more rows in the data.\n",
-    "___"
+    "___\n",
+    "\n",
+    "We currently support the version of Vizzu below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from panel.models.vizzu import VERSION\n",
```

Comment:
It will be almost impossible to use the Vizzu pane without knowing which version of the docs to refer to. So I added the version in the notebook.

Please note 1) I removed the links to the documentation as it gave 404 and I could not find the new links 2) The current version of Vizzu is 0.12.

Reply:
You could use the sphinx hook here https://github.com/holoviz/panel/blob/9404b4348a80a190b3dcda7f270ba3e5b3c10210/doc/conf.py#L238